src/FeatureFlagsUserRoles.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace FriendsOfCat\LaravelFeatureFlags;
+
+interface FeatureFlagsUserRoles
```

Comment:
I guess we could call this interface something more generic, so it's not tight to the roles, and we can later implement a custom method for email, and so on.